### DISCUSSION TITLE
Can't connect to local Raspberry IP from Linux Mint PC for the first time during set up in Docker

### DISCUSSION THREAD

**Discussion by ReaderGuy42** [advplyr/audiobookshelf#2016] [Q&A]
I'm following [this tutorial](https://www.makeuseof.com/host-raspberry-pi-audiobook-library-audiobookshelf/) trying to set up Audiobookshelf.

I'm at the step Setting Up Audiobookshelf From Your Home Network

everything else has worked fine. I did the whole process via SSH from my Linux Mint PC, which worked surprisingly well (don't have much experience with this). As far as I can tell, I successfully installed Audiobookshelf in a Docker.

But now I can't seem to connect to the RasPi's local IP address. I do hostname -I still in SSH and get several numbers. One of them starts with 192 so I put that into the browser on my Mint PC, followed by the port :13378.

And then it either just tells me right away that it is Unable to connect, or once or twice it tells me that it can't connect securely (via https) but I could connect via http, but when I click yes, it just times out too.

Any advice appreciated!

I'm not interested connecting it to the internet, so I really just want to connect to it locally, if that makes a difference.

Thanks!

Edit: I imagine I've found the issue, but don't know how to solve it:

```
docker container ls -a
CONTAINER ID   IMAGE                                   COMMAND                  CREATED         STATUS                       PORTS     NAMES
05a952677529   ghcr.io/advplyr/audiobookshelf:latest   "docker-entrypoint.s…"   2 minutes ago   Exited (133) 2 minutes ago             username_audiobookshelf_1
```

If I'm reading that right, it exits right when I launch it.

Why is that? Or rather, how do I find out why? And would dockers usually start on their own on launch? Or do I have to manually launch it via SSH every time?

**Comment by frogmannnn** [comment]
In the previous step on the tutorial make sure you include the -d (detached) otherwise the container will stop when you close the thread. And yes in the docker compose you can set a restart variable to define what you want it to do. Ie add
`restart: always` at the same indent as `container_name: audiobookshelf`

There are also other options like `unless-stopped`

I suspect the problem here is you docker configuration. Have a look at some general docker / docker compose guides. It is very powerful.
